### ISSUE TITLE
ImageTexture change Size in Editor no 2D preview update 

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#5738]
Changing the Size of an ImageTexture(Resource) in the Editor is not updating the  2D preview(you have to click on the sprite to update).

Adding this solves the issue.

https://github.com/godotengine/godot/blob/master/scene/resources/texture.cpp#L32

```
#include "core_string_names.h"
```

https://github.com/godotengine/godot/blob/master/scene/resources/texture.cpp#L140

```
emit_signal(CoreStringNames::get_singleton()->changed);
```

I am not sure what " _change_notify();" does, maybe this has to be added too.

Is it even wise to change the size of an ImageTexture in the Editor?

I can't see any effect:
 1.The size changes are never saved.
   1.1. Saving the resource after changing the size is only saving the old size.
   1.2. Change the "ImageTexture" Size, press save resource, press save scene, restart project. the ImageTexture has the old size.
   1.3. Exporting to an TileSet with the new size is also using the old size.

If this is something that should not be used is it possible to hide the settings?

**Comment by AlexHolly** [comment]
1.In 3.0 New Image has no effect.

2.Using Load wants a .res file. I don't know how to get such a resource.
When I change the filter to "All Files(*)" and open a .png it crashes godot.


![image](https://user-images.githubusercontent.com/4741886/28344375-2e0a151c-6c23-11e7-90cf-dcf8957523af.png)

**Comment by mhilbrunner** [comment]
This seems to be fixed in Godot 3.0 beta 2. Can't reproduce, all of the above changed. Close?

**Comment by AlexHolly** [comment]
- [x] 2D update preview: I don't know how I changed the size of the Resource I think this was fixed I can't see size properties in ImageTexture/StreamTexture.


- [ ] 1. Is still showing "New Image" when you click it nothing happens what is the purpose?
- [ ] 2. Godot is not crashing anymore but shows many errors so if loading a .png is not allowed maybe remove "All Files(*)" selection?

**Comment by akien-mga** [comment]
We have now entered release freeze for Godot 3.0 and want to focus only on release critical issues for that milestone. Therefore, we're moving this issue to the 3.1 milestone, though a fix may be made available for a 3.0.x maintenance release after it has been tested in the master branch during 3.1 development. If you consider that this issue is critical enough to warrant blocking the 3.0 release until fixed, please comment so that we can assess it more in-depth.
